Source/com/drew/metadata/icc/IccReader.java

```diff
@@ -47,7 +47,7 @@
  * @author Yuri Binev
  * @author Drew Noakes https://drewnoakes.com
  */
-public class IccReader implements JpegSegmentMetadataReader, MetadataReader
```

Comment:
Was there a compelling reason to remove the interface here? I've wondered for a while how useful that interface is, but it might still break users' code.